l10n_br_fiscal/models/comment.py

```diff
@@ -159,12 +158,11 @@ def compute_message(self, vals):
         mako_safe_env = copy.copy(mako_template_env)
         mako_safe_env.autoescape = False
 
-        result = ""
+        comments = [manual_comment] if manual_comment else []
```

Comment:
Fizemos as alterações, a única coisa que mudamos foi por esse if para o manual comment ao invés do laço for dentro do join.